test/HeywoodTest.js

```diff
@@ -0,0 +1,47 @@
+const { expect } = require("chai");
+const chai = require("chai");
+chai.use(require("chai-as-promised"));
+const heywood = require("./../site-scrapers/HeywoodHealthcare");
+const { site } = require("./../site-scrapers/HeywoodHealthcare/config");
+
+describe(`${site.name}`, function () {
+    let context;
+    let page;
+
+    beforeEach(async function () {
+        context = await browser.createIncognitoBrowserContext();
+        page = await browser.newPage();
+    });
+
+    afterEach(async function () {
+        await page.close();
+    });
+
+    it("Test against live site", async function () {
+        await page.goto(site.signUpLink);
+        await expect(heywood(browser)).to.eventually.include({
+            hasAppointments: false,
+            totalAvailability: 0,
```

Comment:
Approved this PR but just wanted to note that this test will fail if there are appointments. Not sure how testing the live site in CI is possible if the condition may change?

Reply:
@zpeyton Good point! I can modify the test so that it passes based on approved conditions.

Reply:
@zpeyton Fixed!